### PR TITLE
[tmux] Initial integration

### PR DESCRIPTION
**Pull request by snizovtsev**
Depends on: tmux/tmux#2448.

tmux is a terminal multiplexer widely used by engineers. It ranked as [2602](https://qa.debian.org/popcon.php?package=tmux) across all debian packages by number of installs. It is even [used](https://cloud.google.com/shell/docs/configuring-cloud-shell#tmux_support) in Google Cloud Shell by default.

The essential part of tmux is escape sequence parser. It accepts input from arbitrary sources, like remote ssh stream or docker sandbox, and converts them into formatted symbol grid. This could be dangerous since the same process handles user interaction and so it has full capabilities.

The work on this fuzzer has already helped to uncover [CVE-2020-27347](https://seclists.org/oss-sec/2020/q4/100) ([OpenBSD 6.8 errata 003](https://marc.info/?l=openbsd-announce&m=160399126725142)).

### CODE REVIEW

**Comment by snizovtsev** [comment]
Hello? Does it fit accepting criteria? The upstream PR is now [ready for merge](https://github.com/tmux/tmux/pull/2448#issuecomment-723879844) but only if it is accepted by oss-fuzz.

**Review by jonathanmetzman — APPROVED** [review]
LGTM
Yes. We accept this project.
I'm personally happy to do this since I use tmux and find it important for my productivity :-) Thanks!

File: projects/tmux/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+RUN apt-get update && apt-get install -y autoconf automake bison pkg-config \
+    libevent-dev ncurses-dev
+
+RUN git clone --depth 1 https://github.com/snizovtsev/tmux.git
```
Comment (jonathanmetzman) [inline projects/tmux/Dockerfile:21]: Let's try to get this switched to the main repo as soon as possible.

**Review by jonathanmetzman — APPROVED** [review]

**Comment by jonathanmetzman** [comment]
Sorry for the delay.

**Comment by nicm** [comment]
Please take security@openbsd.org off the CC for reports from this.

**Pull request merged**